test/index.test.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const Bookshelf = require('bookshelf')(require('knex')({
+  client: 'sqlite3',
+  connection: {
+    filename: './test/test.sqlite3'
+  }
+}));
```

Comment:
is it possible to git ignore the db file and still have a "migration" command to generate it? i dont know if it's any better, but ive always seen `.sqlite` files ignored.